doc/api/process.md

```diff
@@ -1711,15 +1711,15 @@ the script name. These options are useful in order to spawn child processes with
 the same execution environment as the parent.
 
 ```bash
-node --input-type=module --eval 'console.log("foo")' script.js --version
+node --watch-path=./foo --require ./bar.js script.js --version
 ```
 
 Results in `process.execArgv`:
 
 <!-- eslint-disable semi -->
 
 ```js
```

Comment:
While we're at it, we don't need to disable the linter if we display it as a JSON array
````suggestion
```json
````